stories/Atom/Input-group/input_group.scss

```diff
@@ -1,92 +1,111 @@
-@import "../../assets/scss/mixins";
-@import "../../assets/scss/breakpoints";
-@import "../../assets/scss/variables";
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
 
 .label__caption,
 .help__caption,
 .error__caption {
   @extend %paragragh-font-20;
 }
+
 .help__caption {
   font-size: 14px;
 }
+
 .error__caption {
-  font-size: 16px;
   color: $color-red;
+  font-size: 16px;
 }
 
 .input-group {
   .field__image {
     display: inline-block;
     position: relative;
     width: 100%;
+
     &.search {
-      &:before {
+      &::before {
         @extend %field__image;
-        content: "";
+
         background-image: url(#{$img-path-Iconback}/Search_black.svg);
+        content: '';
       }
+
       &.disabledstate {
-        &:before {
-          content: "";
+        &::before {
           background-image: url(#{$img-path-Iconback}/Searchgray.svg);
+          content: '';
         }
       }
     }
   }
+
   .field__setdate {
     position: relative;
   }
+
   .input-type {
-    margin: $spacing-08 0;
     border: 2px solid $color-black;
-    font-size: 16px;
-    width: 100%;
-    height: 46px;
-    padding-left:10px;
     box-sizing: border-box;
+    font-size: $font-size-16;
+    height: 46px;
+    margin: $spacing-08 0;
+    padding: 10px;
+    width: 100%;
+
     &::placeholder {
       color: $color-gray-500;
     }
+
     &.textarea-field {
-      @include devicebreak(small) {
-        padding-top: 15px;
-      }
+      height: auto;
+      max-width: 100%;
     }
+
     &.search-field {
       padding-left: 35px;
       position: relative;
     }
+
     &.focusstate {
       &:focus {
-        box-shadow: 0px 0px 4px #0468b1;
+        box-shadow: 0 0 4px #0468b1;
       }
     }
+
     &.errorstate {
       border: 2px solid $color-red;
     }
+
     &.disabledstate {
       border: 2px solid $color-gray-400;
+
       &::placeholder {
         color: $color-gray-400;
       }
     }
+
     &.date-field {
       background: url(#{$img-path-Iconback}/calender_black.svg) no-repeat left 15px bottom 12px;
+      color: $color-gray-500;
       padding-left: 40px;
+
       &.disabledstate {
         background: url(#{$img-path-Iconback}/datedisabled.svg) no-repeat left 15px bottom 12px;
+        color: $color-gray-400;
         padding-left: 40px;
       }
     }
   }
+
```

Comment:
@pallavikadam-srijan Please only update background-image and color property and not the complete background property